README.md

```diff
@@ -1,182 +1,163 @@
+```js
+3 - 2
+//=> 1
 ```
-
-For now, we're going to explore numbers and strings the most. Later in the curriculum, you'll learn a lot more about the `boolean`, `null` and `undefined` data types.
-
-### Numbers
-
-JavaScript uses numbers just like you would think of them, and we can even use decimal points. Enter the following in console:
-
-``` javascript
-4
-8.0
-16.123
+...and that strings can be added to other strings:
+```js
+"Hello" + ', ' + `world!`
+//=> "Hello, world!"
 ```
 
-In fact, JavaScript treats _all_ numbers as if they have decimal points (even if they don't). Sometimes this can lead to unexpected consequences. Enter the following in console:
-
-```javascript
-4
+Some programming languages are incredibly strict about how data of different types can interact, and they will refuse to compile a program that uses types incorrectly. Other languages are more forgiving. For example, instead of throwing an error when an integer (`3`) is added to a floating-point number (`0.14159`), Ruby will simply convert the integer into a floating-point number and correctly calculate the sum:
+```ruby
+3 + 0.14159
+#=> 3.14159
 ```
 
-If you press enter, you'll see `4`. Simple enough. Now enter the following:
-
-``` javascript
-4.0000000000000001
+JavaScript is a little _too_ nice. No matter what weird combination of types you give it, JavaScript won't throw an error and will return _something_ (though that _something_ might make no sense at all). This runs the gamut from the halfway-defensible...
+```js
+"High " + 5 + "!"
+//=> "High 5!"
 ```
+...to the [comical][Wat]:
+```js
+null ** 2 // null to the power of 2
+//=> 0
 
-If you press `enter`, you'll see... `4`. Hrm. It's not important to know the details of how this behavior works right now (although we encourage you to search for resources online if you find it interesting!), but it is important to know that it happens.
-
-### Strings
-
-Strings are very straightforward in JavaScript. They are collections of characters. Plus signs are used to concatenate strings.
-
-In their most basic form, strings look like
+undefined ** null // undefined to the power of null
+//=> 1
 
-``` javascript
-"I'm a string"
-'I\'m also a string'
+{} + {} // empty object plus empty object
+//=> "[object Object][object Object]" <-- That's a string!
 ```
 
-Notice that we added a `\` before the `'` in the second string above. Because we use quotation marks to tell JavaScript, "Hey, this is a string!", we have to _escape_ quotation marks when they're inside of a string so that JavaScript knows to treat them as part of the string.
+Why JavaScript returns a string when we ask it to add two empty objects is anyone's guess. The language is trying to bend over backwards for its human masters, always returning actionable data instead of throwing errors. However, in its rush to please, JavaScript has created a veritable minefield of type errors that both novice and expert programmers have to keep an eye on.
```

Comment:
As much as I *love* making fun of JS...we are trying to get students excited about JS. I would drop this section. Maybe talking about the benefits (trying to return something for its human masters)